version/version.go

```diff
@@ -0,0 +1,33 @@
+package version
+
+import (
+	"fmt"
+	"os/exec"
+	"runtime"
+	"strings"
+)
+
+var (
+	// Version shows the last bbolt binary version released.
+	Version = "1.3.7"
+)
+
+// PrintVersionInfo prints the information regrading version.
+func PrintVersionInfo() string {
+	return fmt.Sprintf(
+		"bbolt version: %s \nGit SHA: %s \nGo Version: %s \nGo OS/Arch: %s/%s",
+		Version, getGitSHA(), runtime.Version(), runtime.GOOS, runtime.GOARCH,
+	)
+}
+
+// getGitSHA returns the bbolt code commit SHA on git.
+func getGitSHA() string {
+	cmd := exec.Command("git", "rev-parse", "HEAD")
```

Comment:
This command gets executed during run time. It's not going to work when you distribute the binary. I suggest leaving this out unless it's necessary.

Reply:
For ref the approach used in `etcd-io` is to use `ldflags` and include git sha at build time: https://github.com/etcd-io/etcd/blob/main/scripts/build_lib.sh#L18

Reply:
> I suggest leaving this out unless it's necessary.

not necessary, I have removed it now. PTAL

Reply:
> For ref the approach used in etcd-io/etcd is to use ldflags to include git sha at build time

then I have to introduced `scripts` file(s) to build `bbolt` binary, I don't want to make it more complicated for now.